codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerServiceGeneratorV2.kt

```diff
@@ -353,25 +396,62 @@ class ServerServiceGeneratorV2(
                 }
             }
             """,
-            "InternalFailureGenerics" to internalFailureGenerics.join(", "),
-            "InternalFailureFields" to internalFailureFields.join(", "),
-            "NotSetGenerics" to notSetGenerics.join(", "),
             "NotSetFields" to notSetFields.join(", "),
             "Router" to protocol.routerType(),
             "Protocol" to protocol.markerStruct(),
             *codegenScope,
         )
     }
 
+    private fun missingOperationsError(): Writable = writable {
+        rustTemplate(
+            """
+            ##[derive(Debug)]
+            pub struct MissingOperationsError {
+                service_name: &'static str,
```

Comment:
If this error is code generated and not live in the `rust-runtime` crates then I'm guessing we can insert the `service_name` into the actual `Display` implementation via Kotlin templating.

Reply:
Good point - I fixed it in https://github.com/awslabs/smithy-rs/pull/1954/commits/ef089e6a587ed84874f9268636daff6a036f5423